xt/66-cucumber/README.md

```diff
@@ -41,40 +40,40 @@ In separate terminals, you need to run the following commands from the
 root of the development tree for the above to work:
 
 ```sh
- $ starman bin/ledgersmb-server.psgi
- $ phantomjs --webdriver=4422
+ starman bin/ledgersmb-server.psgi
+ phantomjs --webdriver=4422
 ```
 
 # Code structure
 
 The code consists of the following components:
 
- * Feature files (*.feature), grouped by function into the directories
+* Feature files (*.feature), grouped by function into the directories
    numbered as defined above (e.g. `t/66-cucumber/01-basic/`)
- * Step files, which implement function specific steps, in a
+* Step files, which implement function specific steps, in a
    `step_definitions` subdirectory of each numbered directory
    (e.g. `01-basic/step_definitions/`)
- * `pherkin` extensions in `xt/lib/Pherkin/Extension` which implement
-   * `LedgerSMB`: Steps which define application state (provides
+* `pherkin` extensions in `xt/lib/Pherkin/Extension` which implement
+    * `LedgerSMB`: Steps which define application state (provides
      database handle access)
-   * `PageObject`: Steps to access the browser UI, common to most
+    * `PageObject`: Steps to access the browser UI, common to most
      test scripts
- * `PageObject`s (in `xt/lib/`)
-   * implementing access to browser page
+* `PageObject`s (in `xt/lib/`)
+    * implementing access to browser page
      functionality, deriving from `Weasel::Element`, thereby referring
      to their root DOM element
-   * self-registering Weasel::WidgetHandler-s and Weasel::FindExpander-s,
+    * self-registering Weasel::WidgetHandler-s and Weasel::FindExpander-s,
      keeping DOM-tree knowledge local to the PageObject
```

Comment:
Sub lists should only be indented by 2 spaces per level.
Refer to [Github Markdown Syntax Guide](https://guides.github.com/features/mastering-markdown/)

Reply:
All sublists are indented by 1 tab, which is set to 4

Reply:
Be that as it may, the github markdown specification states two space indent for each sublist level.

Reply:
Specs says 1 to 3, I'm ok with standardizing on 2, as shown in most examples